aten/src/ATen/function_wrapper.py

```diff
@@ -832,7 +835,7 @@ def process_option(option, output_options):
         broadcast_arg = get_broadcast_argument(option)
         # "s_" for "same size".
         option['method_prefix_derived'] = '' if broadcast_arg is None else 's_'
-        option['device_guard_declaration'] = device_guard(option, formals)
+        option['device_guard_declaration'] = device_guard(option, formals, False, False)
```

Comment:
boooolean blindness

Reply:
idk why we were defaulting arguments with a single call site.